### PR TITLE
Add some broken catalog-info files

### PR DESCRIPTION
**Pull request by dtuite**
These are useful for testing.

### CODE REVIEW

File: catalog-info-broken.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+metadata:
+  name: sample-service-1
+  description: |
+    A service for testing Backstage functionality. Configured for GitHub Actions and
+    Sentry.
+  annotations:
+    github.com/project-slug: roadiehq/sample-service
+    # The Sentry organization is stored in the app-config.yaml of the Backstage instance.
+    sentry.io/project-slug: sample-service
+  backstage.io/techdocs-ref: dir:./
```
Comment (dtuite) [inline catalog-info-broken.yaml:12]: Invalid indentation.

File: catalog-info-invalid-techdocs.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+metadata:
+  name: sample-service-1
+  description: |
+    A service for testing Backstage functionality. Configured for GitHub Actions and
+    Sentry.
+  annotations:
+    github.com/project-slug: roadiehq/sample-service
+    # The Sentry organization is stored in the app-config.yaml of the Backstage instance.
+    sentry.io/project-slug: sample-service
+    backstage.io/techdocs-ref: dir:./doesnt-exist-docs
```
Comment (dtuite) [inline catalog-info-invalid-techdocs.yaml:12]: Non-existing techdocs location.

File: catalog-info-broken.yaml (hunk 1 above)
Comment (martina-if) [reply catalog-info-broken.yaml:12]: Can we add comments above each line explaining why they are broken?

Comment (martina-if) [inline catalog-info-broken.yaml:12]:
```suggestion
    # Wrong indentation level
  backstage.io/techdocs-ref: dir:./
```

**Comment by iain-b** [comment]
Can this be closed ?

**Pull request closed without merging**